source/jormungandr/jormungandr/street_network/tests/asgard_test.py

```diff
@@ -44,7 +44,7 @@ class FakeInstance(object):
 
     name = 'fake_instance'
     context = None
-    asgard_language = "en-US"
+    language = "en-GB"
```

Comment:
Out of curiousity only: Why did you change that in this test?

Reply:
As discussed: to check that the value is really handled, and also that it's not one of the default values saving the test.